src/services/MobileConnect/index.tsx

```
@@ -0,0 +1,26 @@
+import * as _ from './style';
+import Error from '@/../public/assets/cursor/cursor_block.svg';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)